### PR TITLE
249 - Completing RTL Examples 

### LINKED ISSUE #249: General: Complete RTL Examples for all components
**Describe the bug**
Once the locale PR  https://github.com/infor-design/enterprise-wc/tree/5037-locale is merged this task is to go through and make sure all the current components work in RTL mode.

Accordion and Datagrid are done.

**To Reproduce**

1. See ids-locale mixin docs https://github.com/infor-design/enterprise-wc/blob/5037-locale/src/ids-mixins/README.md
2. Try to see if you can make it work either direction without the mixin
3. If needed add the mixin
4. Add the css needed for the mixin to work

**Expected behavior**
All main pages work in RTL Mode

**Additional context**
Will sync up with you on this one and explain

### PR DESCRIPTION
**Pull request by RiyaTayal20**
**Explain the details for making this change. What existing problem does the pull request solve?**

This PR addresses issues regarding RTL flip in the examples. 

**Related github/jira issue (required)**:

Closes #249 

**Steps necessary to review your pull request (required)**:

- Pull this branch down, build and run the app
- Go to http://localhost:4300/ids-accordion as an example or any of the main page affected controls in this PR 
- Navigate to Inspect Elements
- Switch the language in the ids-container to "ar" and see if all elements on page flip smoothly.
- Then continue to check all the examples.

**Included in this Pull Request**:

### CODE REVIEW

**Comment by RiyaTayal20** [comment]
I was unfortunately unable to finish this PR, but have listed some of the issues that I am aware need to be addressed. 

1. Ids-Badge - In RTL mode one of the badges seems to split and prevent a few of the other badges from flipping to RTL
2. IDS-Checkbox - In RTL mode the checkbox button does not flip to the right side as it should
3. IDS-Pager - In RTL mode the icons need to flip sides as well.
4. IDS-Popup-Menu - In RTL mode the popup menu icon needs to switch sides and transform to point towards the left
5. IDS-Radio - In RTL mode the radio button does not flip to the right side as it should
6. IDS-Spinbox - In RTL mode the sides of the spinbox are hidden when they flip
7. IDS-Switch - In RTL mode the switch button does not flip to the right side as it should
8. IDS-Theme-Switcher - In RTL mode the theme-switcher button padding has to be adjusted 

A lot of these issues relate to one another, so fixing one issue should help in fixing another.

**Comment by rob2d** [comment]
@RiyaTayal20 Thanks on the notes for us here and enjoy the break!

**Comment by RiyaTayal20** [comment]
Thanks so much @rob2d 😄

**Comment by tmcconechy** [comment]
Thanks @RiyaTayal20 i'll finish this up

**Comment by tmcconechy** [comment]
@EdwardCoyle @rob2d @deep7102 I finished up the remaining list and this is ready to review

File: webpack.config.demos.js [hunk 1]
```diff
@@ -33,6 +33,9 @@ module.exports = {
   }, {}),
   devtool: isProduction ? 'cheap-module-source-map' : 'source-map', // try source-map for prod
   mode: isProduction ? 'production' : 'development',
+  infrastructureLogging: {
```
Comment (tmcconechy) [inline webpack.config.demos.js:36]: Reduces log output in the browser

**Review by tmcconechy — APPROVED** [review]

**Review by deep7102 — APPROVED** [review]

**Pull request merged**